examples/src/main/java/org/bitcoinj/examples/ForwardingService.java

```diff
@@ -131,10 +118,9 @@ public Address start() {
      */
     @Override
     public void close() {
-        if (kit.isRunning()) {
-            kit.wallet().removeCoinsReceivedEventListener(listener);
```

Comment:
Where is this listener now removed?

Reply:
It is not explicitly removed, but it is implicitly removed when everything closes.

Reply:
I've added back explicit removal.